Description: Disable the building and running of the test framework
 The test framework is currently disabled because it requires junit 4.7
 specifically and will not run with junit 4.8 currently in unstable
Author: Mat Scales <mat@wibbly.org.uk>
Index: lucene-solr/lucene/build.xml
===================================================================
--- lucene-solr/lucene/build.xml	2011-11-29 09:21:37.509631787 +0000
+++ lucene-solr/lucene/build.xml	2011-11-29 09:21:37.905632895 +0000
@@ -182,7 +182,7 @@
 
   <target name="javadoc" depends="javadocs"/>
   <target name="javadocs" description="Generate javadoc" 
-          depends="javadocs-all, javadocs-core, javadocs-contrib, javadocs-test-framework">
+          depends="javadocs-all, javadocs-core, javadocs-contrib">
     <echo file="${javadoc.dir}/index.html" append="false">
 <![CDATA[<html><head><title>${Name} ${version} Javadoc Index</title></head>
 <body>
@@ -190,7 +190,6 @@
 <ul>
   <li><a href="all/index.html">All</a></li>
   <li><a href="core/index.html">Core</a></li>
-  <li><a href="test-framework/index.html">Test Framework</a></li>
   <li>Contrib packages:</li>
   <ul>
 ]]></echo>
@@ -589,8 +588,7 @@
     </delete>
   </target>
 
-  <target name="build-contrib" depends="compile-test"
-          description="Builds all contrib modules and their tests">
+  <target name="build-contrib" description="Builds all contrib modules and their tests">
     <contrib-crawl target="build-artifacts-and-tests"/>
   </target>
 
Index: lucene-solr/lucene/contrib/analyzers/build.xml
===================================================================
--- lucene-solr/lucene/contrib/analyzers/build.xml	2011-11-29 09:21:25.833599112 +0000
+++ lucene-solr/lucene/contrib/analyzers/build.xml	2011-11-29 09:21:37.905632895 +0000
@@ -63,7 +63,7 @@
     <ant dir="stempel" target="test" />
   </target>
 
-  <target name="build-artifacts-and-tests" depends="default,compile-test" />
+  <target name="build-artifacts-and-tests" depends="default" />
 
   <target name="dist-maven" depends="default">
     <ant dir="common" target="dist-maven" />
Index: lucene-solr/lucene/contrib/contrib-build.xml
===================================================================
--- lucene-solr/lucene/contrib/contrib-build.xml	2011-11-29 09:21:25.793599000 +0000
+++ lucene-solr/lucene/contrib/contrib-build.xml	2011-11-29 09:21:37.909632906 +0000
@@ -60,7 +60,7 @@
   <target name="test" depends="init" if="contrib.has.tests">
     <antcall target="common.test" inheritRefs="true" />
   </target>
-  <target name="build-artifacts-and-tests" depends="jar, compile-test" />
+  <target name="build-artifacts-and-tests" depends="jar" />
 	
   <available 
     type="file" 
Index: lucene-solr/lucene/contrib/xml-query-parser/build.xml
===================================================================
--- lucene-solr/lucene/contrib/xml-query-parser/build.xml	2011-11-29 09:21:25.809599045 +0000
+++ lucene-solr/lucene/contrib/xml-query-parser/build.xml	2011-11-29 09:21:37.909632906 +0000
@@ -43,7 +43,7 @@
   <target name="compile-core" depends="jar-queries,common.compile-core" />
 
   <!-- override contrib-build.xml target to also build web demo -->
-  <target name="build-artifacts-and-tests" depends="jar, compile-test,build-web-demo" />
+  <target name="build-artifacts-and-tests" depends="jar,build-web-demo" />
 
   <target name="default" depends="jar-lucene-core,jar-core,build-web-demo"/>
 
